<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pm_Dialog_Limit</class>
 <widget class="QWidget" name="Pm_Dialog_Limit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/***********************indoor*****************/
#Pm_Dialog_Limit[status = &quot;indoor&quot;]
#widget
{
background-color:rgba(211,211,211,0.9);
}
#Pm_Dialog_Limit[status = &quot;indoor&quot;]
#lblTitle
{
	font: 75 20px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
	line-height:23px;
	font-weight:bold;
}
#Pm_Dialog_Limit[status = &quot;indoor&quot;]
#pb_cancel
{
border-style:flat;
background-color:rgba(0,0,0,0);
background-image: url(:/pm/image/btn_close_normal.png);
}
#Pm_Dialog_Limit[status = &quot;indoor&quot;]
#pb_cancel:pressed
{
background-image: url(:/pm/image/btn_close_click.png);
}
#Pm_Dialog_Limit[status = &quot;indoor&quot;]
#line
{
	background-color:rgba(46,46,46,1);
}
#Pm_Dialog_Limit[status = &quot;indoor&quot;]
QPushButton
{
	color:rgb(130, 130, 130);
	border-style:flat;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#Pm_Dialog_Limit[status = &quot;indoor&quot;]
#pb_LimitTest,
#pb_Edge,
#pb_toggle
{
	background-color:rgba(0,0,0,0);
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#Pm_Dialog_Limit[status = &quot;indoor&quot;]
#pb_LimitTest:checked,
#pb_Edge:checked,
#pb_toggle:checked
{
	padding:7px 44px 6px 25px;
	color:rgba(87, 87, 87, 1);
	border-image: url(:/pm/image/toggle_on.png);
}
#Pm_Dialog_Limit[status = &quot;indoor&quot;]
#pb_LimitTest:!checked,
#pb_Edge:!checked,
#pb_toggle:!checked
{
	padding:7px 17px 6px 36px;
	color:rgba(123, 123, 123, 1);
	border-image: url(:/pm/image/toggle_off.png);
}
#Pm_Dialog_Limit[status = &quot;indoor&quot;]
#pb_Upper:checked
{
	background-color:transparent;
	padding: 8px 14px 7px 15px;
	background-image: url(:/pm/image/fre_checked_left.png);
}
#Pm_Dialog_Limit[status = &quot;indoor&quot;]
#pb_Upper:!checked
{
background-color:transparent;
padding: 8px 14px 7px 15px;
	background-image: url(:/pm/image/fre_checked_right.png);
}

#Pm_Dialog_Limit[status = &quot;indoor&quot;]
#pb_Lower:checked
{
background-color:transparent;
padding: 8px 5px 7px 6px;
	background-image: url(:/pm/image/fre_checked_right.png);
	background-position:right;
}
#Pm_Dialog_Limit[status = &quot;indoor&quot;]
#pb_Lower:!checked
{
background-color:transparent;
padding: 8px 5px 7px 6px;
	background-image: url(:/pm/image/fre_checked_left.png);
	background-position:right;
}
#Pm_Dialog_Limit[status = &quot;indoor&quot;]
#widget&gt;QLabel
{
font: 75 13px &quot;Arial&quot;;
color:rgb(46,46,46);
}
#Pm_Dialog_Limit[status = &quot;indoor&quot;]
Pm_LineEdit
{
	color:rgba(162, 115, 0,1);
	padding:8px 8px 7px 10px ;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#Pm_Dialog_Limit[status = &quot;indoor&quot;]
Pm_LineEdit:focus
{
	outline: none;
	border:2px solid rgba(162,115,0,1);
	padding:6px 6px 5px 8px ;
}
/***********************outdoor*****************/
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#widget
{
background-color:rgba(211,211,211,0.9);
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#lblTitle
{
	font: 75 20px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
	line-height:23px;
	font-weight:bold;
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_cancel
{
border-style:flat;
background-color:rgba(0,0,0,0);
background-image: url(:/pm/image/btn_close_normal.png);
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_cancel:pressed
{
background-image: url(:/pm/image/btn_close_click.png);
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#line
{
	background-color:rgba(46,46,46,1);
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
QPushButton
{
	color:rgba(0, 0, 0,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_LimitTest,
#pb_Edge,
#pb_toggle
{
	background-color:rgba(0,0,0,0);
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_LimitTest:checked,
#pb_Edge::checked,
#pb_toggle:checked
{
	padding:7px 44px 6px 25px;
	color:rgba(87, 87, 87, 1);
	border-image: url(:/pm/image/toggle_on.png);
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_LimitTest:!checked,
#pb_Edge:!checked,
#pb_toggle:!checked
{
	padding:7px 17px 6px 36px;
	color:rgba(123, 123, 123, 1);
	border-image: url(:/pm/image/toggle_off.png);
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_Upper:checked
{
	background-color:transparent;
	padding: 8px 14px 7px 15px;
	background-image: url(:/pm/image/fre_checked_left.png);
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_Upper:!checked
{
background-color:transparent;
padding: 8px 14px 7px 15px;
	background-image: url(:/pm/image/fre_checked_right.png);
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_Lower:checked
{
background-color:transparent;
padding: 8px 5px 7px 6px;
	background-image: url(:/pm/image/fre_checked_right.png);
	background-position:right;
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_Lower:!checked
{
background-color:transparent;
padding: 8px 5px 7px 6px;
	background-image: url(:/pm/image/fre_checked_left.png);
	background-position:right;
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#widget&gt;QLabel
{
	font: 75 13px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
Pm_LineEdit
{
	background-color: rgba(255, 255, 255, 0.6);
	color:rgb(161, 114, 23);
	padding-left:6px;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
Pm_LineEdit:focus
{
	outline: none;
	border:2px solid rgba(162,115,0,1);
}
/***********************night*****************/
#Pm_Dialog_Limit[status = &quot;night&quot;]
#widget
{
background-color:rgba(208,182,139,0.9);
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
#lblTitle
{
	font: 75 20px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
	line-height:23px;
	font-weight:bold;
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
#pb_cancel
{
border-style:flat;
background-color:rgba(0,0,0,0);
border-image: url(:/pm/image/night/btn_close_normal.png);
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
#pb_cancel:pressed
{
border-image: url(:/pm/image/night/btn_close_click.png);
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
#line
{
	background-color:rgba(46,46,46,1);
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
QPushButton
{
	color:rgba(123, 123, 123,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
#pb_LimitTest,
#pb_Edge,
#pb_toggle
{
	background-color:rgba(0,0,0,0);
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
#pb_LimitTest:checked,
#pb_Edge::checked,
#pb_toggle:checked
{
	padding:7px 44px 6px 25px;
	color:rgba(87, 87, 87, 1);
	border-image: url(:/pm/image/toggle_on.png);
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
#pb_LimitTest:!checked,
#pb_Edge:!checked,
#pb_toggle:!checked
{
	padding:7px 17px 6px 36px;
	color:rgba(123, 123, 123, 1);
	border-image: url(:/pm/image/toggle_off.png);
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
#pb_Upper:checked
{
	background-color:transparent;
	padding: 8px 14px 7px 15px;
	background-image: url(:/pm/image/fre_checked_left.png);
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
#pb_Upper:!checked
{
background-color:transparent;
padding: 8px 14px 7px 15px;
	background-image: url(:/pm/image/fre_checked_right.png);
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
#pb_Lower:checked
{
background-color:transparent;
padding: 8px 5px 7px 6px;
	background-image: url(:/pm/image/fre_checked_right.png);
	background-position:right;
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
#pb_Lower:!checked
{
background-color:transparent;
padding: 8px 5px 7px 6px;
	background-image: url(:/pm/image/fre_checked_left.png);
	background-position:right;
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
#widget&gt;QLabel
{
	font: 75 13px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
Pm_LineEdit
{
	background: rgba(251,220,167,0.60);
   color:rgba(159,98,0,1);
	padding:8px 8px 7px 10px ;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#Pm_Dialog_Limit[status  =  &quot;night&quot;] 
Pm_LineEdit:disable
{
	color:rgba(133,133,133,0.6);
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
Pm_LineEdit:focus
{
	outline: none;
	border:2px solid rgba(162,115,0,1);
}</string>
  </property>
  <property name="status" stdset="0">
   <string>night</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>252</width>
     <height>648</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>19</y>
      <width>102</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lblTitle
{
	font: 75 20px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
	line-height:23px;
	font-weight:bold;
}</string>
    </property>
    <property name="text">
     <string>Limit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_cancel">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>10</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_cancel
{
background-color:rgba(0,0,0,0);
background-image: url(:/pm/image/btn_close_normal.png);
}
#pb_cancel:pressed
{
background-image: url(:/pm/image/btn_close_click.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>46</y>
      <width>212</width>
      <height>2</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Limt">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#Pm_Dialog_Limit[status = &quot;indoor&quot;]
#pb_Limt
{
	background-color:rgba(0,0,0,0);
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#Pm_Dialog_Limit[status = &quot;indoor&quot;]
#pb_Limt:checked
{
	padding:7px 44px 6px 25px;
	color:rgba(87, 87, 87, 1);
	border-image: url(:/pm/image/toggle_on.png);
}
#Pm_Dialog_Limit[status = &quot;indoor&quot;]
#pb_Limt:!checked
{
	padding:7px 17px 6px 36px;
	color:rgba(123, 123, 123, 1);
	border-image: url(:/pm/image/toggle_off.png);
}

#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_Limt
{
	background-color:rgba(0,0,0,0);
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_Limt:checked
{
	padding:7px 44px 6px 25px;
	color:rgba(87, 87, 87, 1);
	border-image: url(:/pm/image/toggle_on.png);
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#pb_Limt:!checked
{
	padding:7px 17px 6px 36px;
	color:rgba(123, 123, 123, 1);
	border-image: url(:/pm/image/toggle_off.png);
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
#pb_Limt
{
	background-color:rgba(0,0,0,0);
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
#pb_Limt:checked
{
	padding:7px 44px 6px 25px;
	color:rgba(87, 87, 87, 1);
	border-image: url(:/pm/image/toggle_on.png);
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
#pb_Limt:!checked
{
	padding:7px 17px 6px 36px;
	color:rgba(123, 123, 123, 1);
	border-image: url(:/pm/image/toggle_off.png);
}</string>
    </property>
    <property name="text">
     <string>Off</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblSetLlimit">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>70</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Limit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Full">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>69</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#Pm_Dialog_Limit[status = &quot;indoor&quot;] 
#widget&gt;#pb_Full:checked
{ 
	background-image: url(:/pm/image/fre_checked_left.png); 
}
#Pm_Dialog_Limit[status = &quot;indoor&quot;] 
#widget&gt;#pb_Full:!checked
{color: #7B7B7B;
	background-image: url(:/pm/image/fre_checked_right.png); 
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#widget&gt;#pb_Full:checked
{
	background-image: url(:/pm/image/fre_checked_left.png); 
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#widget&gt;#pb_Full:!checked
{color: #7B7B7B;
	background-image: url(:/pm/image/fre_checked_right.png); 
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
#widget&gt;#pb_Full:checked
{
	background-image: url(:/pm/image/fre_checked_left.png); 
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
#widget&gt;#pb_Full:!checked
{color: #7B7B7B;
	background-image: url(:/pm/image/fre_checked_right.png); 
} 
</string>
    </property>
    <property name="text">
     <string>Full</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">LimitTypeGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pb_200k">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>69</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#Pm_Dialog_Limit[status = &quot;indoor&quot;] 
#widget&gt;#pb_200k:checked
{ 
	background-image: url(:/pm/image/fre_checked_right.png);
	background-position:right;
}
#Pm_Dialog_Limit[status = &quot;indoor&quot;] 
#widget&gt;#pb_200k:!checked
{ color: #7B7B7B;
	background-image: url(:/pm/image/fre_checked_left.png);
	background-position:right;
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#widget&gt;#pb_200k:checked
{ 
	background-image: url(:/pm/image/fre_checked_right.png);
	background-position:right;
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
#widget&gt;#pb_200k:!checked
{ color: #7B7B7B;
	background-image: url(:/pm/image/fre_checked_left.png);
	background-position:right;
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
#widget&gt;#pb_200k:checked
{ 
	background-image: url(:/pm/image/fre_checked_right.png);
	background-position:right;
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
#widget&gt;#pb_200k:!checked
{ color: #7B7B7B;
	background-image: url(:/pm/image/fre_checked_left.png);
	background-position:right;
}</string>
    </property>
    <property name="text">
     <string>200 kHz</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">LimitTypeGroup</string>
    </attribute>
   </widget>
   <widget class="Pm_LineEdit" name="ledForwardUp">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">0.0 dBm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblForwardUp">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>150</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Forward
Upper Limit</string>
    </property>
   </widget>
   <widget class="Pm_LineEdit" name="ledForwardLow">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0 dBm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblForwardLow">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>110</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Forward
Lower Limit</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblReverseLow">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>190</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Reverse
Lower Limit</string>
    </property>
   </widget>
   <widget class="Pm_LineEdit" name="ledReverseUp">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0 dBm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Pm_LineEdit" name="ledReverseLow">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0 dBm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblReverseUp">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>230</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Reverse
Upper Limit</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblVideo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Video BW</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_4k">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>380</y>
      <width>69</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#Pm_Dialog_Limit[status = &quot;indoor&quot;]
QPushButton
{
text-align:center;
padding-left:0px;
color: #7B7B7B;
background-image: url(:/pm/image/btn_mkr_nor.png);
}
#Pm_Dialog_Limit[status = &quot;indoor&quot;]
QPushButton:pressed
{
background-image: url(:/pm/image/btn_mkr_click.png);
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
QPushButton
{
text-align:center;
padding-left:0px;
color: #7B7B7B;
background-image: url(:/pm/image/btn_mkr_nor.png);
}
#Pm_Dialog_Limit[status = &quot;outdoor&quot;]
QPushButton:pressed
{
background-image: url(:/pm/image/btn_mkr_click.png);
}

#Pm_Dialog_Limit[status = &quot;night&quot;]
QPushButton
{
text-align:center;
padding-left:0px;
color: #7B7B7B;
background-image: url(:/pm/image/btn_mkr_nor.png);
}
#Pm_Dialog_Limit[status = &quot;night&quot;]
QPushButton:pressed
{
background-image: url(:/pm/image/btn_mkr_click.png);
}</string>
    </property>
    <property name="text">
     <string>4 kHz</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">LimitTypeGroup</string>
    </attribute>
   </widget>
   <zorder>pb_Limt</zorder>
   <zorder>lblTitle</zorder>
   <zorder>pb_cancel</zorder>
   <zorder>line</zorder>
   <zorder>lblSetLlimit</zorder>
   <zorder>pb_Full</zorder>
   <zorder>pb_200k</zorder>
   <zorder>ledForwardUp</zorder>
   <zorder>lblForwardUp</zorder>
   <zorder>ledForwardLow</zorder>
   <zorder>lblForwardLow</zorder>
   <zorder>lblReverseLow</zorder>
   <zorder>ledReverseUp</zorder>
   <zorder>ledReverseLow</zorder>
   <zorder>lblReverseUp</zorder>
   <zorder>lblVideo</zorder>
   <zorder>pb_4k</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Pm_LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">Pm_lineedit.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="LimitTypeGroup"/>
 </buttongroups>
</ui>
